### DISCUSSION TITLE
Vue prototype no work

### DISCUSSION THREAD

**Discussion by andrespaucar** [vuejs/pinia#1396] [Help and Questions]
I am going from vuex to pinia but I have a problem with the Vue.prototype.$axios

In Vue Main
`Vue.prototype.$axios = axios`

In pinia - not work
` const res = await this.$axios.get('/users/records') `

**Comment by cannap** [comment]
hi maybe you want this 
https://pinia.vuejs.org/core-concepts/plugins.html#adding-new-external-properties not sure

**Comment by andrespaucar** [comment]
Hi, thanks for the help,
add the code like this 
![image](https://user-images.githubusercontent.com/35370608/175832892-1c965e41-ec17-465c-8b81-9221aad6f6a6.png)
![image](https://user-images.githubusercontent.com/35370608/175832902-1433d571-d70f-48c3-a35e-29106587af36.png)
